src/python/grpcio_tests/tests_py3_only/interop/xds_interop_client.py

```diff
@@ -107,6 +172,9 @@ def await_rpc_stats_response(
             for method, count_by_peer in self._rpcs_by_method.items():
                 for peer, count in count_by_peer.items():
                     response.rpcs_by_method[method].rpcs_by_peer[peer] = count
+            if len(self._metadata_by_peer.keys()) > 0:
```

Comment:
Nit: The conditional isn't necessary. If the dictionary is empty, the loop on the following line will not execute.